themes/default/content/docs/get-started/_index.md

```diff
@@ -15,32 +15,25 @@ aliases:
 
 Pulumi is a modern infrastructure as code platform that allows you to use familiar programming languages and tools to build, deploy, and manage cloud infrastructure.
 
-Select one of the following options to deploy a simple application in your cloud using Pulumi:
+We have guides for AWS, Azure, Google Cloud and Kubernetes using TypeScript, JavaScipt, Python, Go or C#.
 
-<div class="tiles flex-wrap mt-4">
-    <div class="pb-4 md:pr-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "/docs/get-started/aws" >}}">
-            <img class="h-10 mx-auto" src="/logos/tech/aws.svg" alt="AWS">
-        </a>
-    </div>
-    <div class="pb-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "/docs/get-started/kubernetes" >}}">
-            <img class="h-10 mx-auto" src="/logos/tech/k8s.svg" alt="Kubernetes">
-        </a>
-    </div>
-    <div class="pb-4 md:pr-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "/docs/get-started/azure" >}}">
-            <img class="h-10 mx-auto" src="/logos/tech/azure.svg" alt="Azure">
-        </a>
-    </div>
-    <div class="pb-4 md:w-1/2">
-        <a class="tile p-8" href="{{< relref "/docs/get-started/gcp" >}}">
-            <img class="h-10 mx-auto" src="/logos/tech/gcp.svg" alt="Google Cloud">
-        </a>
-    </div>
-</div>
+## Deploy an Application Guides
```

Comment:
This is technically fine, but not super engaging, as it's expressed in terms of nouns ("Guides", "AWS Guide") rather than verbs ("Deploy an application", "Deploy to AWS"). As a visitor who's probably just clicked a button with an imperative verb on it ("get started") on a page whose title is also "Get Started", I might be more motivated to keep moving forward if the words kept telling me what I'll _do_ by following one of these guides, rather than just listing them. The navigation buttons and page titles of each guide follow this pattern as well -- install, configure, create, review, etc.

```suggestion
## Deploy an Application
```